torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5468,21 +5468,29 @@ def __init__(self,
 
 
 def sample_inputs_linalg_cholesky_inverse(op_info, device, dtype, requires_grad=False, **kwargs):
-    # Generate Cholesky factors of positive-definite (non-singular) Hermitian (symmetric) matrices
```

Comment:
Does this still apply?

Reply:
Do we keep it? Actually, it would make sense to update the doc on cholesky that the diagonal of a cholesky factor is real and non-negotiable.